Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithBypassDamper.mo

```diff
@@ -0,0 +1,347 @@
+    Modelica.Media.Interfaces.PartialCondensingGases
+    "Supply air";
+  replaceable package Medium2 =
+    Modelica.Media.Interfaces.PartialCondensingGases
+    "Exhaust air";
+  parameter Modelica.Units.SI.MassFlowRate m1_flow_nominal
+    "Nominal supply air mass flow rate";
+  parameter Modelica.Units.SI.MassFlowRate m2_flow_nominal
+    "Nominal exhaust air mass flow rate";
+  parameter Modelica.Units.SI.PressureDifference dp1_nominal = 125
+    "Nominal supply air pressure drop";
+  parameter Modelica.Units.SI.PressureDifference dp2_nominal = 125
+    "Nominal exhaust air pressure drop";
+  parameter Real P_nominal(final unit="W")
+    "Power consumption at the design condition";
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_nominal(
+    final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsLatCoo_nominal(
+    final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenCooPL(
+    final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsLatCooPL(
+    final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenHea_nominal(
+    final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsLatHea_nominal(
+    final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsSenHeaPL(
+    final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsLatHeaPL(
+    final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the heating mode";
+
+  Modelica.Blocks.Interfaces.RealInput uBypDamPos(
+    final unit="1",
+    final min=0,
+    final max=1)
+    "Bypass damper position"
+    annotation (Placement(transformation(extent={{-220,100},{-180,140}}),
+      iconTransformation(extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Interfaces.BooleanInput opeSig
+    "True when the wheel is operating"
+    annotation (Placement(transformation(extent={{-220,-20},{-180,20}}),
+      iconTransformation(extent={{-140,60},{-100,100}})));
+  Modelica.Blocks.Interfaces.RealOutput P(
+    final unit="W")
+    "Electric power consumption"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+  Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses.HeatExchangerWithInputEffectiveness
+    hex(
+    redeclare package Medium1 = Medium1,
+    redeclare package Medium2 = Medium2,
+    final m1_flow_nominal=m1_flow_nominal,
+    final m2_flow_nominal=m2_flow_nominal,
+    final show_T=true,
+    final dp1_nominal=0,
```

Comment:
@SenHuang19 The `dp1_nominal` and `dp2_nominal` are set to be 0 and the pressure drops are set to be on the by pass dampers (and they are 125 Pa by default.)
In the `WheelWithVariableSpeed`, the `dp1_nominal` and `dp2_nominal` are set to be 125 Pa directly on the heat exchanger. Why should we have this difference?

Reply:
125 Pa is a typical pressure drop of a wheel based on [ref](https://c03.apogee.net/mvc/home/hes/land/el?utilityname=duquesnelight&spc=UD&id=1116#:~:text=Heat%20wheels%20have%20a%20relatively,be%20reduced%20in%20some%20cases.) and I set it on the by pass dampers to provide a high damper authority ratio, which can help to achieve a good control performance.
In the <code>WheelWithVariableSpeed</code>, a similar need doesn't exist so I just set the pressure drop to the wheel directly for simplification.
Please let me know if any of them need to be modified.